src/test/kotlin/blackjack/model/DeckTest.kt

```diff
@@ -0,0 +1,25 @@
+package blackjack.model
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+
+class DeckTest {
+    val deck = Deck()
+
+    @Test
+    fun `deck should contain 52 cards`() {
+        assertThat(deck.getCards().count()).isEqualTo(52)
```

Comment:
```suggestion
        assertThat(deck.count()).isEqualTo(52)
```

Why don't we let deck count the cards by itself? 
Let's try our best to take out the value inside of a class only for output purposes. :)

Reply:
I have added a `countCards()` method to `Deck` and updated tests to use it instead of accessing the internal card list directly. 
Thanks for the suggestion! 🙌